kayenta-core/src/main/java/com/netflix/kayenta/metrics/MetricsService.java

```diff
@@ -38,4 +38,9 @@
   default List<Map> getMetadata(String metricsAccountName, String filter) throws IOException {
     return Collections.emptyList();
   }
+
+  List<String> getLocations(String metricsAccountName);
+
+  List<String> getRecommendedLocations(String metricsAccountName);
```

Comment:
Comments on both of these methods would also be helpful since they are only used by Atlas at the moment (esp. for implementors of new metrics service integrations).

Reply:
Will add.